Library/Homebrew/cmd/test-bot.rb

```diff
@@ -91,6 +92,13 @@ def failed?
 
     def puts_command
       cmd = @command.join(" ")
+      if ENV["TRAVIS"]
+        puts cmd
+        @travis_timer_id = rand(2**32).to_s(16)
+        puts "travis_time:start:$travis_timer_id:#{@travis_timer_id}"
+        puts "travis_fold:start:#{@command.join(".")}"
+        return
+      end
       print "#{Tty.blue}==>#{Tty.white} #{cmd}#{Tty.reset}"
```

Comment:
I don't think `==>` looks weird. Currently it output strange empty line. https://travis-ci.org/Homebrew/homebrew/jobs/80815392.
While the old one https://travis-ci.org/Homebrew/homebrew/jobs/80807853 looks much better.